C++/Graphs/StoerWagner.cpp

```diff
@@ -0,0 +1,91 @@
+#include <iostream>
```

Comment:
why did you declare ```iostream```, ```vector``` and ```algorithm``` header-files, since you have already declared ```bits/stdc++```

Reply:
Sir, Microsoft Visual C++ does not support #include<bits/stdc++.h> so i generally write all the header file which is used in code becuase alot of student is using this repo

Reply:
If you want i can remove all other header files

Reply:
Then you can remove the redundant header files